gc_ruboconfig.gemspec

```diff
@@ -9,5 +9,6 @@ Gem::Specification.new do |spec|
 
   spec.files         = 'rubocop.yml'
   spec.add_dependency 'rubocop', '>= 0.70'
+  spec.add_dependency 'rubocop-rails', '>= 2.0.1'
```

Comment:
Do we actually want this - what about where we use `gc_ruboconfig` in non-Rails apps? Sorry if my other suggestion was unclear, I think we should just put this in the readme so people know how to inherit from that other gem :)

Reply:
I suppose that's fair. I don't think we have any non-Rails at the moment, but there's no reason we won't.